pretyping/tacred.ml

```diff
@@ -984,7 +984,7 @@ let e_contextually byhead (occs,c) f = { e_redfun = begin fun env sigma t ->
     if nowhere_except_in && (!pos > maxocc) then (* Shortcut *) t
     else
     try
-      let subst =
+      let (_sigma,subst) = (* FIXME *)
```

Comment:
You can probably simply update the evdref above instead of dropping the new constraints, which will generate anomalies. As duly noted, this code is fishy, no reason to make it even more so.